styles.css

```diff
@@ -1,5 +1,5 @@
 /* MOBILE LAYOUT */
-@media all and (max-width: 750px)
+@media all and (max-width: 699px) {
```

Comment:
This media query is a bit too big for the mobile sizing. Because the cards are percentage based, just below this breakpoint I get an overly stretched card, but at smaller sizes it looks correct. If you want to keep this `max-width`, you could instead use a set width for the cards and/or increase the margin.

![670 px](https://user-images.githubusercontent.com/18703294/114249459-e64de080-9957-11eb-89e7-7eff4567d3a9.jpeg)